api/pkg/cli/formatter/field.go

```diff
@@ -124,10 +124,13 @@ func findSpaceIndexFromLast(str string) int {
 
 // FormatVersion returns version appended with (latest) if the
 // latest field passed is true
-func FormatVersion(version string, latest bool) string {
+func FormatVersion(version string, latest bool, deprecated bool) string {
 	if latest {
 		return version + " (Latest)"
 	}
+	if deprecated {
```

Comment:
Handle the case if latest version is deprecated?

Reply:
yes